README.md

```diff
@@ -113,6 +113,24 @@ windows-*`].
 
 📚 If you're interested in more discussion of alternatives, check out [#4].
 
+## Development
+
+![YAML](https://img.shields.io/static/v1?style=for-the-badge&message=YAML&color=CB171E&logo=YAML&logoColor=FFFFFF&label=)
+![Bash](https://img.shields.io/static/v1?style=for-the-badge&message=Bash&color=4EAA25&logo=GNU+Bash&logoColor=FFFFFF&label=)
+
+This is a GitHub Action, so we inevitably use YAML. Make sure you quote the
+right things! To get started editing, you can use [GitHub.dev] for simple
```

Comment:
I think mentioning github.dev might be a bit too much info. It's a cool feature, but this is leaning into "how to use GitHub" instead of "how to work on *this* project" IMO. Codespaces is one thing because we have a devcontainer configuration set up.